src/kernel.ts

```diff
@@ -1023,6 +1032,7 @@ class Kernel implements IKernel {
   private _commPromises: Map<string, Promise<IComm>> = null;
   private _comms: Map<string, IComm> = null;
   private _isWebSocketClosing = false;
+  private _targetRegistry: { [key: string]: (comm: IComm, msg: IKernelMessage) => void; } = Object.create(null);
```

Comment:
Ah, but `loadClass` is expecting this type of map, so I'll keep it as a regular object.